cmake/alpakaCommon.cmake

```diff
@@ -467,173 +459,49 @@ endif()
 # Find HIP.
 if(ALPAKA_ACC_GPU_HIP_ENABLE)
 
-    if(NOT DEFINED ALPAKA_HIP_VERSION)
-        set(ALPAKA_HIP_VERSION 4.0)
-    endif()
+    # minimal supported HIP version
+    set(_ALPAKA_HIP_MIN_VER 4.0)
```

Comment:
We use the same pattern for boost too.  This will separate defining the minimum number from `find_package()` and increases the readability because it is not only a magic number `find_package(hip 4.0)`.
This is helpful for someone who is not familiar with CMake.